test/cctest/test_inspector_socket_server.cc

```diff
@@ -0,0 +1,517 @@
+#include "inspector_socket_server.h"
+
+#include "node.h"
+#include "gtest/gtest.h"
+
+#include <algorithm>
+#include <sstream>
+
+using InspectorSocketServer = node::inspector::InspectorSocketServer;
+using SocketServerDelegate = node::inspector::SocketServerDelegate;
```

Comment:
I moved the directives inside the namespace. Linter was fine - don't know if it is because of its settings.